### ISSUE TITLE
Images hang and never send

### ISSUE THREAD

**Issue by alexbaldwin** [ZekeSnider/Jared#80]
Checking the Console.app logs and it appears that Jared is receiving my POST command but the photo never actually sends. Are you using this on Ventura? Possible that an update is required for the latest OS?

```
{"body":{"message":"Demo"},"attachments":[{"filePath":"~/Pictures/ab.jpg"}],"recipient":{"handle":"REDACTED"}}
```

When trying the same type of command with `imessage-ruby` it has the same issue.

**Comment by alexbaldwin** [comment]
Was able to send photos, but had to change to an absolute file path, i.e. `/Users/alexbaldwin/Pictures/ab.jpg`. Not sure if this is macOS Venture specific or what.

**Comment by ZekeSnider** [comment]
Yeah this seems like a limitation in the AppleScript API on Ventura or later, since `imessage-ruby` likely uses that as well. In Jared we could expand the tilde automatically, I will tag this as an enhancement ticket to capture that.

**Comment by shortcurb** [comment]
I've done some trial-and-error testing after finding this same bug. I'm not sure whether its an issue with how iMessage handles attachments/picture or some kind of gatekeeping done by Apple's servers to prevent spam/abuse. 

This is what I did to send pictures reliably (although I'm not able to reproduce the issue or re-try the fix):
1. Use absolute paths
2. If images, put them in the /Users/$username/Pictures folder (obv on the computer running Jared)
3. Send to a phone number linked to an iCloud account, rather than an email address linked to an iCloud account

YMMV, I just wanted to share what worked for me.
Lastly, I noticed loading an attachment from the Downloads folder would cause Messages app to treat it as if it were a generic file, rather than an image (with the image preview and other context options)
